kk/models/hearing.py

```diff
@@ -18,7 +24,7 @@ class Hearing(ModifiableModel):
     )
     close_at = models.DateTimeField(verbose_name=_('Closing time'), default=timezone.now)
     n_comments = models.IntegerField(verbose_name=_('Number of comments'), blank=True, default=0)
-    status = models.BooleanField(verbose_name=_('Whether hearing is open'), default=True)
+    status = models.BooleanField(verbose_name=_('Whether hearing is closed'), default=False)
```

Comment:
Should this field just be called `closed` instead of the more nebulous `status`?